test/functional/setup.js

```diff
@@ -1,150 +1,155 @@
+
+var slConnector = null;
+var slBrowsers  = null;
+
+
+function initBrowsersInfo (tc) {
+    browsersInfo = config.browsers
+        .filter(function (browser) {
+            return config.isTravisTask || !browser.remoteOnly;
+        })
+        .map(function (settings) {
+            return {
+                settings:   settings,
+                connection: tc.createBrowserConnection()
+            };
+        });
+}
+
+function openRemoteBrowsers () {
+    slConnector = new SauceLabsConnector(config.sauceLabs.username, config.sauceLabs.accessKey);
+
+    return slConnector
+        .connect()
+        .then(function () {
+            var openBrowserPromises = browsersInfo.map(function (browserInfo) {
+                return slConnector.startBrowser(browserInfo.settings, browserInfo.connection.url,
+                    config.sauceLabs.jobName);
+            });
+
+            return Promise.all(openBrowserPromises);
+        })
+        .then(function (browsers) {
+            slBrowsers = browsers;
+        });
+}
+
+function openLocalBrowsers () {
+    var openBrowserPromises = browsersInfo.map(function (browserInfo) {
+        return browserNatives.getBrowserInfo(browserInfo.settings.alias)
+            .then(function (browser) {
+                return browserNatives.open(browser, browserInfo.connection.url);
+            });
+    });
+
+    return Promise.all(openBrowserPromises);
+}
+
+function closeRemoteBrowsers () {
+    var closeBrowserPromises = slBrowsers.map(function (browser) {
+        return slConnector.stopBrowser(browser);
+    });
+
+    return Promise.all(closeBrowserPromises)
+        .then(function () {
+            return slConnector.disconnect();
+        });
+}
+
+function closeLocalBrowsers () {
+    var closeBrowserPromises = browsersInfo.map(function (browserInfo) {
+        return browserNatives.close(browserInfo.connection.getStatus().url);
+    });
+
+    return Promise.all(closeBrowserPromises);
+}
+
+before(function () {
+    return createTestCafe(config.testCafe.hostname, config.testCafe.port1, config.testCafe.port2)
+        .then(function (tc) {
+            testCafe = tc;
+
+            initBrowsersInfo(tc);
+            site.create(config.site.port1, config.site.port2, config.site.viewsPath);
+
+            if (config.isTravisTask)
+                return openRemoteBrowsers();
+
+            return openLocalBrowsers();
+        })
+        .then(function () {
+            global.testReport = null;
+
+            global.runTests = function (fixture, testName, opts) {
+                var report                = '';
+                var runner                = testCafe.createRunner();
+                var fixturePath           = path.join(path.dirname(caller()), fixture);
+                var skipJsErrors          = opts && opts.skipJsErrors;
+                var quarantineMode        = opts && opts.quarantineMode;
+                var elementAvailabilityTimeout = opts && opts.elementAvailabilityTimeout;
```

Comment:
Can we set it to `200` by default for all tests except those which specify timeout explicitly?